mon-pix/app/pods/components/module/recap/styles.scss

```
@@ -0,0 +1,48 @@
+.module-recap {
+  margin: 0 auto;
+  padding: var(--pix-spacing-4x) var(--pix-spacing-4x) var(--pix-spacing-12x) var(--pix-spacing-4x);
+  background-color: var(--pix-primary-100);
+  border-radius: 16px;
+
+  &__header {
+    margin-bottom: var(--pix-spacing-6x);
+    text-align: right;
+  }
+
+  &__title, &__objectives {
+    margin-bottom: var(--pix-spacing-12x);
+  }
+
+  &__title {
+    @extend %pix-title-l;
+
+    text-align: center;
+  }
+
+  &__link {
+    width: fit-content;
```

Pourquoi cette propriété ?

---

L'implémentation dans Pix UI force le full-width du button-link, d'où ce choix

---

Vincent a trouvé quelque chose à ce sujet : https://stackoverflow.com/questions/71439217/link-a-has-100-width-in-display-flex-button-dont

---

Geoffroy*

---

Côté on a utilisé 

```suggestion
    display: inline-flex;
```

que je trouve plus lisible. Mais j'ai peut être mal compris

---

je viens de tester, ça permet bien d'avoir un lien de la largeur de son contenu mais il se retrouve justifié à gauche et je n'arrive pas à le re-forcer en centré

---

En fait comme chaque composant commun on devrais le wrapper pour pas se faire piéger. Ça devrait te donner plus de contrôle